tests/common/cisco_data.py

```diff
@@ -35,20 +35,12 @@ def get_markings_dut(duthost, key_list=['ecn_dequeue_marking', 'ecn_latency_mark
     local_file = contents['dest']
     with open(local_file) as fd:
         json_contents = json.load(fd)
-    required_entry = None
-    for i in range(len(json_contents['devices'])):
-        try:
-            json_contents['devices'][i].get('id')
-            required_entry = i
-        except KeyError:
-            continue
-
-    if required_entry is None:
-        raise RuntimeError("Couldnot find the required entry(id) in the config file:{}".format(config_file))
-    original_values = {}
+    markings_dict = {}
+    # Getting markings from first device.
+    device = json_contents['devices'][0]
```

Comment:
Is it guaranteed that json_contents['devices'] will have only one entry ?

Reply:
In non-T2 asics, it should have only 1.
If it has more than one, they should all be in sync, and it's only necessary to return the first.